std/math.d

```diff
@@ -328,6 +335,9 @@ template floatTraits(T)
             enum EXPPOS_SHORT = 0;
             enum SIGNPOS_BYTE = 0;
         }
+
+        import std.typecons : Tuple;
+        alias Integral = Tuple!(ulong, "significand", ushort);
```

Comment:
Both the docs at the top of the module and static asserts on lines 212-227 seem to imply that 80-bit `real`s are only supported on little-endian systems. Furthermore, AFAICT, all of the code currently dealing with those assumes little-endiannes (which is why I added `significand` in the first place).